src/main/resources/stubs/encoding/binary/binary.gobra

```diff
@@ -44,106 +44,110 @@ type ByteOrder interface {
 // type littleEndian struct{}
 // type bigEndian struct{}
```

Comment:
Couldn't this be removed?

Reply:
This is the original definition. I would like to keep this here for when we add support to global variables

Reply:
I'm fine with keeping the `var`s. I was more referring to the `type` declarations as you seem to use `int` instead of empty interfaces.
Why don't you use empty interfaces? Is this still because of the problematic encoding of fields of finite cardinality (as mentioned in the changeset below)?

Reply:
No, it is because empty interfaces are not valid types for constants in Go. Empty structs are an acceptable type but initializing a value of struct{} cannot be done statically and thus, the compiler rejects it

Reply:
I see thanks